ilan/build

```diff
@@ -21,15 +21,13 @@ if [ $# != 3 ]; then
   exit 1
 fi
 
-docker build -t installation-manager im || exit $?
-docker run --rm -v $(pwd):/host installation-manager /host/install_ihs $1 $2 $3 || exit $?
+docker build -t installation-manager . --build-arg VERSION=$1 --build-arg IBM_ID=$2  --build-arg IBM_PASSWORD=$3|| exit $?
```

Comment:
install_ihs.sh is called within docker during image build.  The images are "cached up to the point of line 47. so there's no need to concern about installing Installation Manager again.